src/styles/_themes.scss

```diff
@@ -157,6 +163,12 @@
   --form-input-border: #{color(base-light)};
   --form-input-placeholder: #{$theme-white};
 
+  // Pagination
+  --pagination-page-number-bg: #{$white};
```

Comment:
Suggestion: I don't see pagination in figma but the white bg on the current pagination seems strange in dark mode. I wonder if it should be the green or something. Confirm with @malworks on what color the bg should be here.

Reply:
Good catch. Pagination is under "For Eng: Dark Theme 1.0" in Figma down at the very bottom, but I did have the color wrong. Updated!